cmd/skr/main.go

```diff
@@ -41,7 +41,7 @@ func main() {
 	httpPort := flag.String("port", "8080", "Port on which to listen")
 	allowTestingMismatchedTCB := flag.Bool("allowTestingMismatchedTCB", false, "For TESTING purposes only. Corrupts the TCB value")
 	// NOTE: these 4 input arguments are typically only used in AKS, not ACI
-	grpcPort := flag.String("keyprovider_sock", "127.0.0.1:50000", "Port on which the grpc key provider to listen")
+	grpcPort := flag.String("keyprovider_sock", os.Getenv("KEYPROVIDER_SOCK"), "Port on which the grpc key provider to listen")

```

Comment:
This has a host address as well as the port. If we don’t enforce exactly over that then the host could be set to 0.0.0.0 and then be available outside of the UVM. If you look at the comments (approx line 88) about taking the host as part of the command line you will see that we previously decided that that was too much of a risk. If people want to do that they will have to make the change there and build a bespoke container.

Also, the passing of this as an individual env variable directly to the binary rather than to the skr.sh seems to be a mismatch of style. That leads to another question about whether it is ever that case that both the http and grpc endpoints will be in use at once. If not, then maybe better to use the existing port variable and signal the grpc vs http by some other means.

Reply:
I agree with the first paragraph, I'll make changes so KEYPROVIDER_SOCK only takes in a port instead of an address and port.

For the styling, I'm working on a PR that will let users use capital snake case e.g. LOG_FILE (which is more common for env vars) instead of the less conventional casing of LogFile. It'll still have backwards compatibility but it'll let users directly use `/bin/skr` more easily, instead of having to go through `skr.sh`

Reply:
Updated so env var casing is staying consistent with what is already present in the repo. There is no longer plans for another PR to change the casing for existing environment variables